tests/validations.sh

```diff
@@ -288,15 +288,54 @@ spk service install-build-pipeline --org-name $AZDO_ORG -u $remote_repo_url -d $
 pipeline_created=$(az pipelines show --name $frontend_pipeline_name --org $AZDO_ORG_URL --p $AZDO_PROJECT)
 
 # Verify frontend service pipeline run was successful
-verify_pipeline_with_poll $AZDO_ORG_URL $AZDO_PROJECT $frontend_pipeline_name 300 15
+verify_pipeline_with_poll $AZDO_ORG_URL $AZDO_PROJECT $frontend_pipeline_name 300 15 1
 
 echo "Finding pull request that $frontend_pipeline_name pipeline created..."
 approve_pull_request $AZDO_ORG_URL $AZDO_PROJECT "Updating $FrontEnd image tag to master"
 # --------------------------------
 
+##################################
+# App Mono Repo create ring
+##################################
+ring_name=test
+
+cd $TEST_WORKSPACE
+cd $mono_repo_dir
+
+echo "Create ring"
+spk ring create $ring_name
+git add -A
+git commit -m "Adding test ring"
+git push -u origin --all
+
+# Wait for the lifecycle pipeline to finish and approve the pull request
+verify_pipeline_with_poll $AZDO_ORG_URL $AZDO_PROJECT $lifecycle_pipeline_name 180 15 2
```

Comment:
Why are we expecting 2 pipelines? Even if we are it seems only one of them will produce a PR. Is that correct?

Reply:
it's the number of builds inside the pipeline, not the number of pipelines